tests/data/dimension_associations/scenario__subsector.csv

```diff
@@ -0,0 +1,14 @@
+scenario,subsector
+baseline,FullServiceRestaurant
+baseline,Hospital
+baseline,LargeHotel
+baseline,LargeOffice
+baseline,MediumOffice
+baseline,Outpatient
+baseline,PrimarySchool
+baseline,QuickServiceRestaurant
+baseline,StandaloneRetail
+baseline,StripMall
+baseline,SmallHotel
+baseline,SmallOffice
+baseline,Warehouse
```

Comment:
This seems redundant with the scenario,data_source and data_source,subsector associations. Is that what's intended?